controllers/bootstrap/init.go

```diff
@@ -180,6 +180,15 @@ func (b *Bootstrap) InitResources(instance *apiv3.CommonService) error {
 		}
 	}
 
+	// Check storageClass
+	csStorageClass, err := util.GetStorageClass(b.Reader)
+	if err == nil {
+		if err := util.ValidateStorageClass(csStorageClass); err != nil {
+			klog.Errorf("StorageClass is not found in current cluster")
+			return err
```

Comment:
I think it will be better if we could combine those two lines together to reduce the duplicated info in log, for example directly return the error, and in func `ValidateStorageClass`
```
if size <= 0 {
    return fmt.Errorf("StorageClass is not found in current cluster")
}
```

then the output of log will be
```
StorageClass Number: 0
Failed to initialize resources: StorageClass is not found in current cluster
Failed to reconcile ibm-common-services/common-service: StorageClass is not found in current cluster
```